test/harmony.js

```diff
@@ -2548,6 +2548,24 @@ data = {
         }
     },
 
+    'Harmony object pattern single rest-element property': {
+        '{...a}': {
+            generateFrom: {
```

Comment:
I would prefer to see a whole Program here that contains the ObjectPattern in an allowed position.

Reply:
Ok. Expect update soon

Reply:
Updated 5b635e6